### DISCUSSION TITLE
Pushin Website

### DISCUSSION THREAD

**Discussion by ghost** [GrapesJS/grapesjs#5597] [Q&A] [answered]
Hey, grapeusers, I would like to publish my website that I've been working on for months now. How do I do that? I don't see an option for it like I'm used to with a GUI. I know I can export the code for it, but I wanted to make sure there's no other way before doing that. Thanks for the help :)

**Comment by artf** [comment, marked as answer]
If you're referring to GrapesJS Studio, unfortunately, there is no way to do it via UI but I hope to add it soon with some free providers (eg. Github Pages, Netlify, etc.)
